src/components/Recommended/Recommended.css

```diff
@@ -1,22 +1,43 @@
 .recommendedContainer {
     height: 10%;
-    width: 70%;
-    background-color: #865dff;
+    width: 100%;
+    background-color: #191725;
     display: flex;
     flex-direction: column;
     align-items: center;
-    border-color: black;
-    border: solid;
     padding: 10px;
     color: white;
+    font-family: 'quantico'
+}
+
+.recommendedContainer h2 {
+    margin: 1em;
 }
 
 .album {
     display: flex;
     flex-direction: column;
     align-items: center;
+    margin-bottom: 10%;
+    font-size: x-large;
```

Comment:
This is messing up your container width on smaller screens. Remove the font size completely, I think it's too large on smaller screens anyways. Then move it into the media query and add it to `.album p` This should fix the issue of different size containers.

Reply:
I'm not seeing any issues on container width on smaller screens on my end - what exactly are you seeing?

Reply:
oh you mean